app/views/charges/show.html.erb

```diff
@@ -1,7 +1,7 @@
-<%= form_tag cart_path do %>
+<%= form_tag charges_path do %>
 <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
     data-key="<%=ENV['PUBLISHABLE_KEY'] %>"
     data-description="Payment for food"
     data-amount="1000"
     data-locale="auto"></script>
-<% end %>
\ No newline at end of file
+<% end %>
```

Comment:
Out of scope IMO

Reply:
Yes it is, we are taking it out from this PR and making a separate PR for it 👍